pep/app/pods/components/search/item/template.hbs

```diff
@@ -0,0 +1,45 @@
+<li class="mb-4 d-flex justify-content-between">
+    <div>
+        <a href="" {{on "click" (fn @openResultPreview @item)}}>
+            {{@item.documentRef}}
+        </a>
+        {{#if @item.kwicList}}
+            <ul class="font-size-secondary">
+                {{#each @item.kwicList as |searchHit|}}
+                    <li>{{html-safe searchHit}}</li>
+                {{/each}}
+            </ul>
+        {{/if}}
+    </div>
+    <div class="pl-2 d-flex justify-content-end align-items-start">
+        {{#if @item.relatedrx}}
+            <span class="mr-2">
+                <FaIcon @icon="books" class="p-0 "/>
```

Comment:
seems like the right side might be getting overly icon heavy with all 3 icons (related docs, favorites, read later) aligned to the right. It might be nice to move this related documents indicator over to the left, inline and directly to the right of the documentRef text. Also since its its not an action/button, it would better distinguish it from favorite/read later.